src/engine-scripts/puppet/onReady.js

```diff
@@ -26,8 +26,8 @@ module.exports = async ( page, scenario ) => {
 		await require( './collapsedTocState' )( page, hashtags );
 	}
 
-	if ( hashtags.includes( '#scroll-toc' ) ) {
-		await require( './scrollToc.js' )( page );
+	if ( hashtags.includes( '#toggle-toc-subsections' ) ) {
+		await require( './toggleTocSubsections' )( page );
```

Comment:
could we also remove the scroll logic from toggleTocSubsections?

Reply:
Yup, it's already been removed in a prior commit

Reply:
ah i see, i didn't see that thanks